src/Sylius/Bundle/CoreBundle/Fixture/Factory/ChannelExampleFactory.php

```diff
@@ -47,16 +49,21 @@ class ChannelExampleFactory extends AbstractExampleFactory implements ExampleFac
     /** @var OptionsResolver */
     private $optionsResolver;
 
+    /** @var TaxonRepositoryInterface */
+    private $taxonRepository;
+
     public function __construct(
         ChannelFactoryInterface $channelFactory,
         RepositoryInterface $localeRepository,
         RepositoryInterface $currencyRepository,
-        RepositoryInterface $zoneRepository
+        RepositoryInterface $zoneRepository,
+        TaxonRepositoryInterface $taxonRepository
```

Comment:
```suggestion
        ?TaxonRepositoryInterface $taxonRepository = null
```
and probably add `@trigger_error()` that not passing this service is deprecated since 1.8

Reply:
absolutely 👍